.vscode/settings.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
these two files in .vscode will conflict with individual settings won't they? I am not too sure what will happen coz they have been added to the repo but they are in `.gitignore`. May I ll just try to checkout this branch.

Reply:
They should not conflict because of the `.gitignore` and could be used as examples for new comers.
But maybe one needs to be careful about the update if it gets merged.